src/index.js

```diff
@@ -0,0 +1,54 @@
+import './style.css';
+import moreImage from './more.svg';
+
+const listWrapper = document.getElementById('list-wrapper');
+
+const todos = [{
```

Comment:
- [ ] It would be better if we display the list of our todos on page load in our todos array to the web browser. Currently, we have no todos to show